src/config/ts-node.ts

```diff
@@ -8,6 +9,9 @@ import {Debug} from './util'
 // eslint-disable-next-line new-cap
 const debug = Debug('ts-node')
 
+const TYPE_ROOTS = [`${__dirname}/../node_modules/@types`]
```

Comment:
I see these paths are used by `ts-node`, will this work on Windows too or should it use `path.join|sep`?

Reply:
I think so. This was taken from [oclif/config](https://github.com/oclif/config/blob/main/src/ts-node.ts#L50) which as far I know never had windows path issues